sdk/core/core-amqp/src/retry.ts

```diff
@@ -175,7 +175,12 @@ export async function retry<T>(config: RetryConfig<T>): Promise<T> {
   let success = false;
   const totalNumberOfAttempts = config.retryOptions.maxRetries + 1;
   for (let i = 1; i <= totalNumberOfAttempts; i++) {
-    logger.verbose("[%s] Attempt number: %d", config.connectionId, config.operationType, i);
+    logger.verbose(
```

Comment:
Felt like a nice simple change that I can use to ensure I can clone, build, run, and PR 😄 

I couldn't find tests for log output, but let me know if I missed them

Reply:
This should be enough!